src/serac/physics/solid_functional.hpp

```diff
@@ -380,7 +380,7 @@ class SolidFunctional<order, dim, Parameters<parameter_space...>, std::integer_s
           // stress with the shape function gradients.
           //
           // TODO: fix the residual implementation and remove this transpose.
-          return serac::tuple{material.density * d2u_dt2, transpose(flux)};
+          return serac::tuple{material.density * d2u_dt2 * det(I + dp_dX), transpose(flux)};
```

Comment:
It's good that you fixed this, but I don't like that we don't have a test that can tell that it's right. It's  not reasonable to address that in this PR, but I don't want to lose track of it. Maybe I should split up the issues so that we can close the part that is done. Looks like I have to strike the part about `dp_dx` anyway.